R/dataframe-frame.R

```diff
@@ -10,7 +10,7 @@
 #' This function is basically a shortcut for `list(...) |> as_polars_df()`,
 #' so each argument in `...` is converted to a Polars Series by [as_polars_series()]
 #' and then passed to [as_polars_df()].
-#' @aliases polars_data_frame DataFrame
```

Comment:
Why do you remove this?

Reply:
I don't think this is useful and it's not used consistently across docs. I can add it back if you prefer having it, but I just never use aliases (I don't even know how users can see the list of aliases)

Reply:
Ok, so that is limited to `pl__DataFrame`, `pl__LazyFrame`, `pl__Series` and the expr page? Or you wanted to add aliases to other functions as well?

Reply:
Yes, my intention was to be able to use help with typical class names (series, expr, dataframe, ...).
So I don't think alias is necessary for functions/methods.

Reply:
Ok I'll revert this change